python/paddle/trainer/config_parser.py

```diff
@@ -1105,6 +1105,37 @@ def parse_conv(conv, input_layer_name, conv_conf):
             1 + int(math.ceil((2 * conv.padding + conv_conf.img_size \
             - conv.filter_size) / float(conv.stride)))
 
+
+def parse_conv_trans(conv, input_layer_name, conv_conf, num_filters):
```

Comment:
done for the parse_conv_trans
However for the ConvTransLayerBase, I am not sure I can do the same thing since it has a decorator and the overlapping of the codes is not that big.